pypika/terms.py

```diff
@@ -66,10 +67,31 @@ def tables_(self) -> set["Table"]:
     def fields_(self) -> set["Field"]:
         return set(self.find_(Field))
 
+    if TYPE_CHECKING:  # pragma: nocoverage
+
+        @typing.overload
+        @staticmethod
+        def wrap_constant(val: NodeT, wrapper_cls: Type[Term] | None = None) -> NodeT: ...
+        @typing.overload
+        @staticmethod
+        def wrap_constant(val: None, wrapper_cls: Type[Term] | None = None) -> "NullValue": ...
+        @typing.overload
+        @staticmethod
+        def wrap_constant(val: list, wrapper_cls: Type[Term] | None = None) -> "Array": ...
+        @typing.overload
+        @staticmethod
+        def wrap_constant(val: tuple, wrapper_cls: Type[Term] | None = None) -> "Tuple": ...
+        @typing.overload
+        @staticmethod
+        def wrap_constant(val: Any, wrapper_cls: None = None) -> "ValueWrapper": ...
```

Comment:
This one is not like the others.

Reply:
I really don't like such two functions(wrap_constant/wrap_json), they looks strange. It will be very great if you can refactor them...